.github/workflows/pr_ci.yml

```diff
@@ -3,7 +3,7 @@ on:
   pull_request:
     branches: [ main, release/** ]
 env:
-  DOTNET_SKIP_FIRST_TIME_EXPERIENCE: true
+  DOTNET_NOLOGO: true
```

Comment:
Are you sure only this is needed? [The docs](https://learn.microsoft.com/en-us/dotnet/core/tools/dotnet-environment-variables) doesn't say `DOTNET_SKIP_FIRST_TIME_EXPERIENCE` is deprecated, and from the descriptions is seems to me that `DOTNET_NOLOGO` is a subset of `DOTNET_SKIP_FIRST_TIME_EXPERIENCE`.

Reply:
https://github.com/OrchardCMS/OrchardCore/actions/runs/3258551770/jobs/5350727783

![image](https://user-images.githubusercontent.com/171892/196050059-91808ab6-b48d-4454-9587-9f82d4b5c491.png)

Reply:
Thanks!